### ISSUE TITLE
Localization 

### ISSUE THREAD

**Issue by xDangles** [CloudNetService/CloudNet-v3#1398]
### Describe the new feature

Add more keys for CloudNet localization. Not all elements of CloudNet are described in detail and many points cannot be translated. For example, if you write the command "modules", it will display a list of commands with no description. I suggest to make a description for such commands. And this is the minimum I can offer.
![изображение](https://github.com/CloudNetService/CloudNet/assets/70214692/543aec4d-2e1d-43b0-9cc9-9bae45405171)

Also missing are keys for many commands, bugs, and so on. For example "Name(s), Description, Permissions)" and so on.

### Why do you need this feature?

This will help a lot of people get used to CloudNet faster. For example, I had 2-3 developers constantly asking questions about commands, such as what they do and how to use them. And documentation may not always be convenient, complete and understandable in the native language.

### Alternatives

It's simple. It won't take much time for you to add new keys, and the localization community will be pleased.

### Other

Also, if at all possible, it's better to separate console and gaming. In the console it looks just fine, but in the game it's a mess. Although in game it is very convenient to interact with cloudnet, but unpleasant in terms of design, I would like to change it from the configuration to my taste.

### Issue uniqueness

- [X] Yes, this issue is unique. There are no similar issues.

**Comment by 0utplay** [comment]
I really like the idea of making more messages translatable. Are u willing to do a pull request?

I would start with making the titles of all these different tables translatable.

**Comment by xDangles** [comment]
Yes, I've already helped you do the translation on crowdin. I will be happy to support translation into Russia.
